Tried to add desktop-launcher by editing Makefile.in
--- a/Makefile.in
+++ b/Makefile.in
@@ -113,8 +113,8 @@
     || { echo " ( cd '$$dir' && rm -f" $$files ")"; \
          $(am__cd) "$$dir" && rm -f $$files; }; \
   }
-am__installdirs = "$(DESTDIR)$(guimupdocdir)"
-DATA = $(guimupdoc_DATA)
+am__installdirs = "$(DESTDIR)$(guimupdocdir)" "$(DESTDIR)$(guimupappdir)" 
+DATA = $(guimupdoc_DATA) $(guimupapp_DATA)
 RECURSIVE_CLEAN_TARGETS = mostlyclean-recursive clean-recursive	\
   distclean-recursive maintainer-clean-recursive
 AM_RECURSIVE_TARGETS = $(RECURSIVE_TARGETS:-recursive=) \
@@ -297,7 +297,15 @@
 	INSTALL\
 	NEWS
 
-EXTRA_DIST = $(guimupdoc_DATA)
+guimupappdir = ${prefix}/share/applications
+guimupapp_DATA = \
+	usr/share/applications/guimup.desktop\
+	usr/share/pixmaps/guimup32.png\
+	usr/share/pixmaps/guimup48.png\
+	usr/share/pixmaps/guimup64.png
+
+
+EXTRA_DIST = $(guimupdoc_DATA) $(guimupapp_DATA)
 all: config.h
 	$(MAKE) $(AM_MAKEFLAGS) all-recursive
 
@@ -382,6 +390,28 @@
 	files=`for p in $$list; do echo $$p; done | sed -e 's|^.*/||'`; \
 	dir='$(DESTDIR)$(guimupdocdir)'; $(am__uninstall_files_from_dir)
 
+install-guimupappDATA: $(guimupapp_DATA)
+	@$(NORMAL_INSTALL)
+	@list='$(guimupapp_DATA)'; test -n "$(guimupappdir)" || list=; \
+	if test -n "$$list"; then \
+	  echo " $(MKDIR_P) '$(DESTDIR)$(guimupappdir)'"; \
+	  $(MKDIR_P) "$(DESTDIR)$(guimupappdir)" || exit 1; \
+	fi; \
+	for p in $$list; do \
+	  if test -f "$$p"; then d=; else d="$(srcdir)/"; fi; \
+	  echo "$$d$$p"; \
+	done | $(am__base_list) | \
+	while read files; do \
+	  echo " $(INSTALL_DATA) $$files '$(DESTDIR)$(guimupappdir)'"; \
+	  $(INSTALL_DATA) $$files "$(DESTDIR)$(guimupappdir)" || exit $$?; \
+	done
+
+uninstall-guimupappDATA:
+	@$(NORMAL_UNINSTALL)
+	@list='$(guimupapp_DATA)'; test -n "$(guimupappdir)" || list=; \
+	files=`for p in $$list; do echo $$p; done | sed -e 's|^.*/||'`; \
+	dir='$(DESTDIR)$(guimupappdir)'; $(am__uninstall_files_from_dir)
+
 # This directory's subdirectories are mostly independent; you can cd
 # into them and run `make' without going through this Makefile.
 # To change the values of `make' variables: instead of editing Makefiles,
@@ -712,6 +742,9 @@
 	for dir in "$(DESTDIR)$(guimupdocdir)"; do \
 	  test -z "$$dir" || $(MKDIR_P) "$$dir"; \
 	done
+	for dir in "$(DESTDIR)$(guimupappdir)"; do \
+	  test -z "$$dir" || $(MKDIR_P) "$$dir"; \
+	done
 install: install-recursive
 install-exec: install-exec-recursive
 install-data: install-data-recursive
@@ -764,7 +797,7 @@
 
 info-am:
 
-install-data-am: install-guimupdocDATA
+install-data-am: install-guimupdocDATA install-guimupappDATA 
 
 install-dvi: install-dvi-recursive
 
@@ -810,7 +843,7 @@
 
 ps-am:
 
-uninstall-am: uninstall-guimupdocDATA uninstall-local
+uninstall-am: uninstall-guimupdocDATA uninstall-guimupappDATA uninstall-local
 
 .MAKE: $(RECURSIVE_CLEAN_TARGETS) $(RECURSIVE_TARGETS) all \
 	ctags-recursive install-am install-strip tags-recursive
@@ -831,12 +864,13 @@
 	maintainer-clean maintainer-clean-generic mostlyclean \
 	mostlyclean-generic mostlyclean-libtool pdf pdf-am ps ps-am \
 	tags tags-recursive uninstall uninstall-am \
-	uninstall-guimupdocDATA uninstall-local
+	uninstall-guimupdocDATA uninstall-local \
+	uninstall-guimupappDATA install-guimupappDATA
 
 
 # Remove doc directory on uninstall
 uninstall-local:
-	-rm -r $(guimupdocdir)
+	-rm -r $(guimupdocdir) $(guimupappdir)
 
 # Tell versions [3.59,3.63) of GNU make to not export all variables.
 # Otherwise a system limit (for SysV at least) may be exceeded.
