Remove (un)install targets for `${prefix}/doc/guimup`
--- a/Makefile.in
+++ b/Makefile.in
@@ -113,8 +113,7 @@
     || { echo " ( cd '$$dir' && rm -f" $$files ")"; \
          $(am__cd) "$$dir" && rm -f $$files; }; \
   }
-am__installdirs = "$(DESTDIR)$(guimupdocdir)"
-DATA = $(guimupdoc_DATA)
+
 RECURSIVE_CLEAN_TARGETS = mostlyclean-recursive clean-recursive	\
   distclean-recursive maintainer-clean-recursive
 AM_RECURSIVE_TARGETS = $(RECURSIVE_TARGETS:-recursive=) \
@@ -123,7 +122,7 @@
 ETAGS = etags
 CTAGS = ctags
 DIST_SUBDIRS = $(SUBDIRS)
-DISTFILES = $(DIST_COMMON) $(DIST_SOURCES) $(TEXINFOS) $(EXTRA_DIST)
+DISTFILES = $(DIST_COMMON) $(DIST_SOURCES) $(TEXINFOS) 
 distdir = $(PACKAGE)-$(VERSION)
 top_distdir = $(distdir)
 am__remove_distdir = \
@@ -288,16 +287,7 @@
 top_builddir = @top_builddir@
 top_srcdir = @top_srcdir@
 SUBDIRS = src 
-guimupdocdir = ${prefix}/doc/guimup
-guimupdoc_DATA = \
-	README\
-	COPYING\
-	AUTHORS\
-	ChangeLog\
-	INSTALL\
-	NEWS
 
-EXTRA_DIST = $(guimupdoc_DATA)
 all: config.h
 	$(MAKE) $(AM_MAKEFLAGS) all-recursive
 
@@ -360,27 +350,6 @@
 
 distclean-libtool:
 	-rm -f libtool config.lt
-install-guimupdocDATA: $(guimupdoc_DATA)
-	@$(NORMAL_INSTALL)
-	@list='$(guimupdoc_DATA)'; test -n "$(guimupdocdir)" || list=; \
-	if test -n "$$list"; then \
-	  echo " $(MKDIR_P) '$(DESTDIR)$(guimupdocdir)'"; \
-	  $(MKDIR_P) "$(DESTDIR)$(guimupdocdir)" || exit 1; \
-	fi; \
-	for p in $$list; do \
-	  if test -f "$$p"; then d=; else d="$(srcdir)/"; fi; \
-	  echo "$$d$$p"; \
-	done | $(am__base_list) | \
-	while read files; do \
-	  echo " $(INSTALL_DATA) $$files '$(DESTDIR)$(guimupdocdir)'"; \
-	  $(INSTALL_DATA) $$files "$(DESTDIR)$(guimupdocdir)" || exit $$?; \
-	done
-
-uninstall-guimupdocDATA:
-	@$(NORMAL_UNINSTALL)
-	@list='$(guimupdoc_DATA)'; test -n "$(guimupdocdir)" || list=; \
-	files=`for p in $$list; do echo $$p; done | sed -e 's|^.*/||'`; \
-	dir='$(DESTDIR)$(guimupdocdir)'; $(am__uninstall_files_from_dir)
 
 # This directory's subdirectories are mostly independent; you can cd
 # into them and run `make' without going through this Makefile.
@@ -706,12 +675,10 @@
 	       exit 1; } >&2
 check-am: all-am
 check: check-recursive
-all-am: Makefile $(DATA) config.h
+all-am: Makefile config.h
 installdirs: installdirs-recursive
 installdirs-am:
-	for dir in "$(DESTDIR)$(guimupdocdir)"; do \
-	  test -z "$$dir" || $(MKDIR_P) "$$dir"; \
-	done
+
 install: install-recursive
 install-exec: install-exec-recursive
 install-data: install-data-recursive
@@ -764,7 +731,7 @@
 
 info-am:
 
-install-data-am: install-guimupdocDATA
+install-data-am: 
 
 install-dvi: install-dvi-recursive
 
@@ -810,7 +777,7 @@
 
 ps-am:
 
-uninstall-am: uninstall-guimupdocDATA uninstall-local
+uninstall-am: uninstall-local
 
 .MAKE: $(RECURSIVE_CLEAN_TARGETS) $(RECURSIVE_TARGETS) all \
 	ctags-recursive install-am install-strip tags-recursive
@@ -824,19 +791,18 @@
 	distuninstallcheck dvi dvi-am html html-am info info-am \
 	install install-am install-data install-data-am install-dvi \
 	install-dvi-am install-exec install-exec-am \
-	install-guimupdocDATA install-html install-html-am \
+	install-html install-html-am \
 	install-info install-info-am install-man install-pdf \
 	install-pdf-am install-ps install-ps-am install-strip \
 	installcheck installcheck-am installdirs installdirs-am \
 	maintainer-clean maintainer-clean-generic mostlyclean \
 	mostlyclean-generic mostlyclean-libtool pdf pdf-am ps ps-am \
 	tags tags-recursive uninstall uninstall-am \
-	uninstall-guimupdocDATA uninstall-local
+	uninstall-local
 
 
 # Remove doc directory on uninstall
 uninstall-local:
-	-rm -r $(guimupdocdir)
 
 # Tell versions [3.59,3.63) of GNU make to not export all variables.
 # Otherwise a system limit (for SysV at least) may be exceeded.
